src/sys/node/node-worker-thread.ts

```diff
@@ -1,14 +1,21 @@
 import type * as d from '../../declarations';
 
-export const initNodeWorkerThread = (prcs: NodeJS.Process, msgHandler: d.WorkerMsgHandler) => {
+/**
+ * Initialize a worker thread, setting up various machinery for managing
+ * communication between the child process (worker) and the main thread.
+ *
+ * @param process a NodeJS process
+ * @param msgHandler a worker message handler, which processes incoming messages calling
+ */
+export const initNodeWorkerThread = (process: NodeJS.Process, msgHandler: d.WorkerMsgHandler) => {
   const sendHandle = (err: NodeJS.ErrnoException) => {
     if (err && err.code === 'ERR_IPC_CHANNEL_CLOSED') {
-      prcs.exit(0);
+      process.exit(0);
     }
   };
 
   const errorHandler = (stencilMsgId: number, err: any) => {
-    const errMsgBackToMain: d.MsgFromWorker = {
+    const errMsgBackToMain: d.MsgFromWorker<any> = {
```

Comment:
Similar to my question above regarding using `any` here. Feel free to ignore if the answer is the same